pkgs/development/lua-modules/overrides.nix

```diff
@@ -428,6 +429,34 @@ with prev;
   });
 
 
+  liblpeg = prev.lpeg.overrideAttrs (oa: {
+    pname = "liblpeg";
+
+    # https://github.com/neovim/neovim/pull/25315
+    # Original author uses gcc for darwin to compile lpeg onto `.so`.
+    # This causes some issue with the common darwin setup to use clang, which
+    # expects `.dylib` for `CMAKE_SHARED_LIBRARY_SUFFIX`
+    buildPhase = ''
+      sed -i makefile -e "s/CC = gcc/CC = clang/"
+      sed -i makefile -e "s/-bundle/-dynamiclib/"
+      make macosx
+    '';
+
+    installPhase = ''
+      mkdir -p $out/lib
+      mv lpeg.so $out/lib/lpeg.dylib
+    '';
+
+    nativeBuildInputs = [ fixDarwinDylibNames ] ++ oa.nativeBuildInputs;
+  });
+
+  lpeg = prev.lpeg.overrideAttrs (oa: {
+    postInstall = ''
+      mkdir -p $out/lib
+      cp -r ${final.liblpeg}/lib/* $out/lib/
+    '';
+  });
```

Comment:
Not sure if this is necessary if it's only needed for Neovim.
Btw, this is what's causing `nixpkgs-review` to build so many packages.

Reply:
This is indeed needed for Neovim to only bump nixpkgs up. We could just not override lpeg and have neovim's `buildInputs` contain `liblpeg` explicitly

Reply:
neovim is required to test the vimPlugins so any changes to it will trigger rebuilds. It's more a matter of lpeg referencing liblpeg which I find confusing.